README.md

```diff
@@ -0,0 +1,14 @@
+![](https://github.com/openmainframeproject/artwork/raw/master/projects/softwarediscoverytool/softwarediscoverytool-color.svg)
+
+# Data for Software Discovery Tool
+  
+![License](https://img.shields.io/github/license/openmainframeproject/software-discovery-tool)
+
+## Overview
+
+Software Discovery Tool is a tool for discovering open source software for zArchitecture/s390x for any Z operating system. The data for the softwares such as thier names and major versions, are contained in this repository. Please refer to the documentation in [this](https://github.com/openmainframeproject/software-discovery-tool) repo to learn more about the data directory structure.
```

Comment:
A little grammar and spelling fix up:

The data for the software, such as their names and versions available, are contained in this repository.